media_driver/linux/common/ddi/media_libva_putsurface_linux.h

```diff
@@ -31,7 +31,7 @@
 #include <va/va_dricommon.h>
 #include "mos_defs.h"
 
-#define LIBVA_X11_NAME "libva-x11.so"
+#define LIBVA_X11_NAME "libva-x11.so.2"
```

Comment:
what's the libva version scheme? How often it could be updated?

Reply:
Read here:
https://github.com/intel/libva/blob/d6fd111e2062bb4732db8a05ed55fc01771087b4/configure.ac#L55

Shortly, libva*.so.x.y, where x=API_MAJOR+1. That's why we now have .so.2 and it will remain the same as long as we will not want to completely refactor LibVA again.